src/Codeception/Module/Asserts.php

```diff
@@ -60,7 +60,7 @@ public function expectThrowable($throwable, callable $callback): void
      * Check if the given throwable matches the expected data,
      * fail (throws an exception) if it does not.
      */
```

Comment:
please add a `@param` annotation to this method's docblock to indicate that this may be `int|Exception`.

Reply:
@TavoNiievez your suggestion doesn't make sense to me, the code can be of literally any type (see https://www.php.net/manual/en/throwable.getcode.php)